docs/v1.13.x/native-dom.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Native DOM Helpers
```

Comment:
Suggest `Triggering Native vs. jQuery DOM Events` as the title of this page. Would be nice to have something as expressive as possible so that people can easily realize "oh, yeah...that sounds like what I'm dealing with."

Reply:
Seems like it's a good direction. I'm finally [agree](https://github.com/san650/ember-cli-page-object/pull/339#issuecomment-344063047) with you that the key thing here are Events